doc/releases/release-notes-2.7.rst

```diff
@@ -166,18 +166,56 @@ Architectures
 Bluetooth
 *********
 
+* Updated all APIs and internal implementation to be conformant with the new
+  inclusive terminology in version 5.3 of the Bluetooth Core Specification
+
 * Audio
 
+  * Added the Microphone Input Control Service and client.
+  * Split the configuration options into connected and broadcast groups
+  * Added support for a new sent callback to be notified when an SDU has been
+    transmitted
```

Comment:
It's a good question whether this should be under "Audio", under "Host" or in its own separate "Isochronous" grouping, as these two changes were for ISO, and ISO != Audio, but merely a feature used by LE Audio.

Reply:
As an additional comment for ISO, I think this misses the rather significant change to the ISO API (as the text here only mentions configurations options). 

I'd add something like 
` * Change connected isochronous API to use group-based opaque struct`

Reply:
>It's a good question whether this should be under "Audio", under "Host" or in its own separate "Isochronous" grouping, as these two changes were for ISO, and ISO != Audio, but merely a feature used by LE Audio.

For the sake of simplicity in the release notes, let's group those under Audio, since there are no other users of ISO outside of audio today?

Reply:
>As an additional comment for ISO, I think this misses the rather significant change to the ISO API (as the text here only mentions configurations options).

Added this now